website/source/docs/enterprise/index.html.md

```diff
@@ -9,24 +9,55 @@ description: |-
-for reads and scheduling, and hot server failover on a per availability zone basis. See the 
-links below for a detailed overview of each feature.
-
-- [Namespaces](/docs/enterprise/namespaces/index.html)
-- [Resource Quotas](/docs/enterprise/quotas/index.html)
-- [Sentinel Policies](/docs/enterprise/sentinel/index.html)
-- [Preemption](/docs/enterprise/preemption/index.html)
-- [Advanced Autopilot](/docs/enterprise/autopilot/index.html)
-
-Click [here](https://www.hashicorp.com/go/nomad-enterprise) to set up a demo or request a trial 
-of Nomad Enterprise.
+Nomad Enterprise adds collaboration, operational, and governance capabilities to Nomad.  Nomad Enterprise is available as a base Platform package with an optional Governance & Policy add-on module.
+
+Please navigate the sub-sections for more information about each package and its features in detail.
+
+## Nomad Enterprise Platform
+Nomad Enterprise Platform enables operators to easily upgrade Nomad as well as enhanced performance and availability through Advanced Autopilot features such as Automated Upgrades, Enhanced Read Scalability, and Redundancy Zones.
+
+### Automated Upgrades
+Automated Upgrades allows operators to deploy a complete cluster of new servers and then simply wait for the upgrade to complete. As the new servers join the cluster, server logic checks the version of each Nomad server node. If the version is higher than the version on the current set of voters, it will avoid promoting the new servers to voters until the number of new servers matches the number of existing servers at the previous version. Once the numbers match, Nomad will begin to promote new servers and demote old ones.
+
+See the [Autopilot - Upgrade Migrations](https://www.nomadproject.io/guides/operations/autopilot.html#upgrade-migrations) documentation for a thorough overview.
+
+### Enhanced Read Scalability
+This feature enables an operator to introduce non-voting server nodes to a Nomad cluster. Non-voting servers will receive the replication stream but will not take part in quorum (required by the leader before log entries can be committed). Adding explicit non-voters will scale reads and scheduling without impacting write latency.
+
+See the [Autopilot - Read Scalability](https://www.nomadproject.io/guides/operations/autopilot.html#server-read-and-scheduling-scaling) documentation for a thorough overview.
+
+### Redundancy Zones
+Redundancy Zones enables an operator to deploy a non-voting server as a hot standby server on a per availability zone basis. For example, in an environment with three availability zones one can run one voter and one non-voter in each availability zone, for a total of six servers. If an availability zone is completely lost, only one voter will be lost, so the cluster remains available. If a voter is lost in an availability zone, Nomad will promote the non-voter to voter automatically, putting the hot standby server into service quickly.
+
```

Comment:
"Nomad will promote the non-voter to voter" -> "Nomad will promote the non-voter to a voter"